Common/Securities/SecurityTransactionManager.cs

```diff
@@ -332,6 +364,18 @@ public List<Order> GetOpenOrders(Func<Order, bool> filter = null)
             return _orderProcessor.GetOpenOrders(x => filter(x));
         }
 
+        /// <summary>
+        /// Gets open orders matching the specified filter. Specifying null will return an enumerable
+        /// of all open orders.
```

Comment:
> Specifying null will return an enumerable of all open orders.

This will probably blow up? since `GetOpenOrders(Func<Order, bool> filter = null)` covers no filter case, don't think we need to support users sending `None`?